src/safe_dicom_loader.py

```diff
@@ -0,0 +1,55 @@
+"""
+This file is used to open the dicom image, if it cant be opened it will throw errors
+"""
+
+import numpy as np
+from dicom_utils import numpy_to_qimage
+
+def dicom_image_opener(ds):
```

Comment:
I think the function name is a bit misleading.
Maybe something that indicates that you are getting a Qimage from DICOM data.